plugins/tiddlywiki/highlight-legacy/usage.tid

```diff
@@ -0,0 +1,16 @@
+title: $:/plugins/tiddlywiki/highlight/usage
+
+! Usage
+
+Fenced code blocks can have a language specifier added to trigger highlighting in a specific language. Otherwise heuristics are used to detect the language.
+
+```
+ ```js
+ var a = b + c; // Highlighted as JavaScript
+ ```
+```
+! Adding Themes
```

Comment:
Just noticed that "Adding Themes" is here in the "usage" tiddler, which does indeed suggest that "Adding Languages" should be here too.

Reply:
done